### PR TITLE
Remove Flake Tracker and Setup Detox 18 in Wallet Repo

### PR DESCRIPTION
**Pull request by MuckT**
### Description

This PR removes the [Flake Tracker](https://github.com/celo-org/celo-monorepo/tree/master/packages/flake-tracker) from detox e2e tests, and moves to config management into the wallet repo.

### Other changes

- Set log levels on detox to info.
- Increased the threshold for detox debug-synchronization to 5 seconds.
- Various Config and file changes to support moving configuration management all into the wallet repo.
- Add `https://blockchain-api-dot-celo-mobile-alfajores.appspot.com` to detox [device.setURLBlacklist](https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#deviceseturlblacklisturls)
- Adjusted Celo Education steps that were failing in CI.

### Tested

- Tests ran locally on iOS and Android

```Bash
# Navigate to packages/mobile
cd packages/mobile

# Run on iOS
yarn test:e2e:ios

# Run on Android
yarn test:e2e:android
```

### How others should test

Run the tests locally & observe the results in the CICD pipeline.

### Backwards compatibility

This PR only impacts e2e tests and existing test run without error.

### CODE REVIEW

File: packages/mobile/e2e/init.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
 beforeAll(async () => {
   await device.launchApp({
+    newInstance: false,
     permissions: { notifications: 'YES', contacts: 'YES' },
   })
-})
-
-beforeEach(async () => {
+  await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-alfajores.appspot.com.*'])
```
Comment (jeanregisser) [inline packages/mobile/e2e/init.js:9]: What does this do? could be nice to add a small comment explaining it.

**Review by jeanregisser — APPROVED** [review]
Looks great! 👍 
Would be nice to add in the description why we are removing Flake Tracker.
From what I see it's because it can't work correctly with this type of test where if you retry you'd be still at the end state of the first try which failed. Hence no change for the initial conditions to be met again.

File: package.json [hunk 2]
```diff
@@ -54,6 +54,7 @@
     "@types/jest": "^26.0.20",
     "babel-jest": "^26.6.3",
     "codecov": "^3.6.5",
+    "detox": "^18.18.1",
```
Comment (etuleu) [inline package.json:57]: why move this to the root package?

Comment (MuckT) [reply package.json:57]: This was needed to get detox working correctly with the change of removing the flake tracker. I had tried keeping it in the mobile package.json and was having issues. Moving it to the root of the repo resolved these issues.

File: packages/mobile/e2e/init.js (hunk 1 above)
Comment (MuckT) [reply packages/mobile/e2e/init.js:9]: This prevents detox sync form waiting on requests to the listed API. This API is called every time the user lands on the transactions home screen, so removing the waiting increases the speed of the tests.

**Pull request merged**